app/interactors/authorize_request.rb

```diff
@@ -3,6 +3,13 @@ class AuthorizeRequest
   include FailOnError
 
   def call
-    Services::DomainOwnershipService.new.authorize!(context.identity, context.request)
+    context.request.fqdns.each do |fqdn|
```

Comment:
I'm a little ambivalent as to whether this logic belongs here or in the service class, with the AuthReq class serving as a wrapper that delegates authing to a set of services as needs arise. What do you think?

Reply:
I was thinking about "Service" as interface to external service whereas in this case it's just logic pertaining to the local models (the external service call moved to `RefreshDomain`)

Reply:
Okay, I'm fine with leaving it here for now, but if it becomes multiple cases next time we should split it out...